ratis-server/src/main/java/org/apache/ratis/server/impl/ConfigurationManager.java

```diff
@@ -42,10 +42,10 @@ public class ConfigurationManager {
    * the last entry of the map. Otherwise is initialConf.
    */
   @SuppressWarnings({"squid:S3077"}) // Suppress volatile for generic type
-  private volatile RaftConfigurationImpl currentConf;
+  private RaftConfigurationImpl currentConf;
   /** Cache the peer corresponding to {@link #id}. */
   @SuppressWarnings({"squid:S3077"}) // Suppress volatile for generic type
-  private volatile RaftPeer currentPeer;
+  private RaftPeer currentPeer;
```

Comment:
Please remove also the `@SuppressWarnings({"squid:S3077"})` for both fields.  It was for the `volatile` [false positive](https://community.sonarsource.com/t/rule-s3077-frequently-generates-a-false-positive/27479).

Reply:
Ok!